src/global.json

```
@@ -0,0 +1,7 @@
+﻿{
+  "sdk": {
+    "version": "8.0.401",
```

```suggestion
    "version": "8.0.400",
```
The lastest version of Visual Studio only installs 8.0.400.  Let's use 8.0.400 or a prior version.  (I was running 8.0.302 a few minutes ago.)

---

8.0.401 is a security patch